Tasks/MavenV3/mavenutil.ts

```diff
@@ -33,7 +33,9 @@ async function convertXmlStringToJson(xmlContent: string): Promise<any> {
 
 function writeJsonAsXmlFile(filePath: string, jsonContent: any, rootName:string): Q.Promise<void> {
     let builder = new xml2js.Builder({
-        pretty: true,
+        renderOpts:{
+            pretty: true,
+        },
```

Comment:
According to the [documentation](https://www.npmjs.com/package/xml2js#options-for-the-builder-class), `pretty` is a nested `renderOpts` setting. If in the future the typings for this package will be added, this could cause a compilation error.
Make sense to remove this option, because the `pretty` options is `true` by default.